org.audacityteam.Audacity.json

```diff
@@ -1,19 +1,18 @@
 {
     "app-id": "org.audacityteam.Audacity",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.24",
+    "runtime-version": "3.26",
     "sdk": "org.gnome.Sdk",
     "command": "audacity",
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
-        "--filesystem=host"
+        "--filesystem=home"
```

Comment:
Honestly leaving host probably makes sense. I doubt Audacity will ever support portals or anything.

Reply:
I remember something failing to launch when giving access host (I think it was scribus) so I just subconsciously use home, but I’ll change it back.